user-access-event-handlers/src/main/java/no/unit/nva/useraccess/events/service/EchoMigrationService.java

```diff
@@ -1,10 +1,10 @@
-package no.unit.nva.useraccess.events;
+package no.unit.nva.useraccess.events.service;
 
 import java.util.ArrayList;
 import java.util.List;
 import no.unit.nva.useraccessmanagement.model.UserDto;
 
-public class EchoMigrationService implements MigrationService {
+public class EchoMigrationService implements UserMigrationService {
```

Comment:
This is a simple service for (online human) testing purposes to be sure that the scanner works as expected. This class will be deleted very soon. There is no dependency on this class in the tests.